src/Class/DecEq/Instances/Extra.agda

```diff
@@ -0,0 +1,26 @@
+{-# OPTIONS --safe #-}
+
+-- Extra instances for decidable equality.
+module Class.DecEq.Instances.Extra where
+
+open import Data.Refinement
+  using (Refinement; value; _,_)
+open import Function
+  using (_∘_)
+open import Relation.Binary.PropositionalEquality
+  using (cong; refl)
+open import Relation.Nullary.Decidable
+  using (no; yes)
+
+open import Class.DecEq
+  using (DecEq; _≟_)
+
+-- Equality for a Refinement type is decide if the equality
+-- for the type to be refined is decidable.
+DeqEq-Refinement
```

Comment:
It'd be fantastic to upstream this to https://github.com/agda/agda-stdlib-classes

Reply:
Ok, but not in the present pull request in order to keep the scope limited.

I have created an issue https://github.com/agda/agda-stdlib-classes/issues/17